test/e2e/pods.go

```diff
@@ -289,29 +291,33 @@ var _ = Describe("Pods", func() {
 		pods, err := podClient.List(labels.SelectorFromSet(labels.Set(map[string]string{"time": value})), fields.Everything())
 		Expect(len(pods.Items)).To(Equal(1))
 
-		By("retrieving the pod")
-		podOut, err := podClient.Get(pod.Name)
-		if err != nil {
-			Fail(fmt.Sprintf("Failed to get pod: %v", err))
-		}
-
-		By("updating the pod")
-		value = "time" + value
-		pod.Labels["time"] = value
-		pod.ResourceVersion = podOut.ResourceVersion
-		pod.UID = podOut.UID
-		pod.Spec.Host = podOut.Spec.Host
-		pod, err = podClient.Update(pod)
-		if err != nil {
-			Fail(fmt.Sprintf("Failed to update pod: %v", err))
-		}
+		// Standard get, update retry loop
+		expectNoError(wait.Poll(time.Second*5, time.Minute*2, func() (bool, error) {
+			By("updating the pod")
+			value = strconv.Itoa(time.Now().Nanosecond())
+			pod.Labels["time"] = value
+			pod, err = podClient.Update(pod)
+			if err == nil {
+				Logf("Successfully updated pod")
+				return true, nil
+			}
+			if errors.IsConflict(err) {
+				Logf("Conflicting update to pod, re-get and re-update: %v", err)
+				pod, err = podClient.Get((*pod).Name)
```

Comment:
Yes, sorry, and I just remembered I forgot to change the time periods.
Let me change those and get back to you.